kura/test/org.eclipse.kura.wire.component.provider.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.eclipse.kura.wire.component.provider.test
```

Comment:
Wouldn't it be good if we provide more user friendly name to `Bundle-Name`? (such as `Wire Component Provider Tests`)